From: SeongJae Park <sj@kernel.org>
Date: Thu, 10 Oct 2024 09:58:50 -0700
Subject: [PATCH] ==== docs hotfix ====

Docs/mm/damon/maintainer-profile: a couple of minor hotfixes

DAMON maintainer-profile.rst file patches[1] that merged into the v6.12-rc1
have a couple of minor mistakes.  Fix those.

[1] https://lore.kernel.org/20240826015741.80707-1-sj@kernel.org

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/r12Mup0h | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/r12Mup0h

diff --git a/damon_meta_changes/r12Mup0h b/damon_meta_changes/r12Mup0h
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

